<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>main_ui</class>
 <widget class="QWidget" name="main_ui">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>580</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgb(255, 255, 255);
border: 2px solid gray;
border-radius: 10px;
padding: 0 8px;
selection-background-color: darkgray;	</string>
  </property>
  <widget class="QLabel" name="img_label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>0</y>
     <width>131</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;resource\exicom_logo.png&quot; width=&quot;120&quot; height=&quot;40&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="submit_btn">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>10</y>
     <width>75</width>
     <height>37</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>60</width>
     <height>37</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(255, 255, 255);
border: 2px solid gray;
border-radius: 10px;
padding: 0 8px;
selection-background-color: darkgray;
font: 75 15pt &quot;Times New Roman&quot;;</string>
   </property>
   <property name="text">
    <string>Submit</string>
   </property>
   <property name="iconSize">
    <size>
     <width>25</width>
     <height>16</height>
    </size>
   </property>
   <property name="shortcut">
    <string>Return</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>561</width>
     <height>511</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>2</number>
    </property>
    <property name="topMargin">
     <number>5</number>
    </property>
    <property name="rightMargin">
     <number>2</number>
    </property>
    <property name="bottomMargin">
     <number>15</number>
    </property>
    <item row="1" column="0">
     <widget class="QLabel" name="pcb_name">
      <property name="maximumSize">
       <size>
        <width>230</width>
        <height>100</height>
       </size>
      </property>
      <property name="sizeIncrement">
       <size>
        <width>50</width>
        <height>0</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Times New Roman</family>
        <pointsize>20</pointsize>
        <weight>9</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(255, 255, 255);
border: 2px solid gray;
border-radius: 10px;
padding: 0 8px;
selection-background-color: darkgray;
font: 75 20pt &quot;Times New Roman&quot;;</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <property name="text">
       <string>PCB  Name</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="date_label">
      <property name="maximumSize">
       <size>
        <width>230</width>
        <height>100</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Times New Roman</family>
        <pointsize>20</pointsize>
        <weight>9</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(255, 255, 255);
border: 2px solid gray;
border-radius: 10px;
padding: 0 8px;
selection-background-color: darkgray;
font: 75 20pt &quot;Times New Roman&quot;;</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <property name="text">
       <string>Date</string>
      </property>
      <property name="textFormat">
       <enum>Qt::PlainText</enum>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="date_entry">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="maximumSize">
       <size>
        <width>300</width>
        <height>50</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Times New Roman</family>
        <pointsize>15</pointsize>
        <weight>9</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(255, 255, 255);
border: 2px solid gray;
border-radius: 10px;
padding: 0 8px;
selection-background-color: darkgray;
font: 75 15pt &quot;Times New Roman&quot;;</string>
      </property>
      <property name="placeholderText">
       <string>Date</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="serial_label">
      <property name="maximumSize">
       <size>
        <width>230</width>
        <height>100</height>
       </size>
      </property>
      <property name="sizeIncrement">
       <size>
        <width>50</width>
        <height>0</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Times New Roman</family>
        <pointsize>20</pointsize>
        <weight>9</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(255, 255, 255);
border: 2px solid gray;
border-radius: 10px;
padding: 0 8px;
selection-background-color: darkgray;
font: 75 20pt &quot;Times New Roman&quot;;</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <property name="text">
       <string>Serial</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="serial_entry">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="maximumSize">
       <size>
        <width>300</width>
        <height>50</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Times New Roman</family>
        <pointsize>14</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(255, 255, 255);

border: 2px solid gray;
border-radius: 10px;
padding: 0 8px;
selection-background-color: darkgray;
font: 14pt &quot;Times New Roman&quot;;</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QComboBox" name="code_combo">
      <property name="maximumSize">
       <size>
        <width>300</width>
        <height>50</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Times New Roman</family>
        <pointsize>15</pointsize>
        <weight>9</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="focusPolicy">
       <enum>Qt::ClickFocus</enum>
      </property>
      <property name="contextMenuPolicy">
       <enum>Qt::ActionsContextMenu</enum>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(255, 255, 255);
border: 2px solid gray;
border-radius: 10px;
padding: 0 8px;
selection-background-color: darkgray;
font: 75 15pt &quot;Times New Roman&quot;;</string>
      </property>
      <property name="locale">
       <locale language="English" country="India"/>
      </property>
      <property name="currentText">
       <string/>
      </property>
      <property name="maxVisibleItems">
       <number>100</number>
      </property>
      <property name="maxCount">
       <number>100</number>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QLineEdit" name="oper_label">
      <property name="maximumSize">
       <size>
        <width>300</width>
        <height>50</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Times New Roman</family>
        <pointsize>15</pointsize>
        <weight>9</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>IBeamCursor</cursorShape>
      </property>
      <property name="statusTip">
       <string>Name</string>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(255, 255, 255);
border: 2px solid gray;
border-radius: 10px;
padding: 0 8px;
selection-background-color: darkgray;
font: 75 15pt &quot;Times New Roman&quot;;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="dragEnabled">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="fault_obser_label">
      <property name="maximumSize">
       <size>
        <width>230</width>
        <height>100</height>
       </size>
      </property>
      <property name="sizeIncrement">
       <size>
        <width>50</width>
        <height>0</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Times New Roman</family>
        <pointsize>20</pointsize>
        <weight>9</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(255, 255, 255);
border: 2px solid gray;
border-radius: 10px;
padding: 0 8px;
selection-background-color: darkgray;
font: 75 20pt &quot;Times New Roman&quot;;</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <property name="text">
       <string>Fault Observed</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QComboBox" name="fault_combo">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="maximumSize">
       <size>
        <width>300</width>
        <height>50</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Times New Roman</family>
        <pointsize>15</pointsize>
        <weight>9</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(255, 255, 255);
border: 2px solid gray;
border-radius: 10px;
padding: 0 8px;
selection-background-color: darkgray;
font: 75 15pt &quot;Times New Roman&quot;;</string>
      </property>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QLabel" name="remark_label">
      <property name="maximumSize">
       <size>
        <width>230</width>
        <height>100</height>
       </size>
      </property>
      <property name="sizeIncrement">
       <size>
        <width>50</width>
        <height>0</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Times New Roman</family>
        <pointsize>20</pointsize>
        <weight>9</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(255, 255, 255);
border: 2px solid gray;
border-radius: 10px;
padding: 0 8px;
selection-background-color: darkgray;
font: 75 20pt &quot;Times New Roman&quot;;</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <property name="text">
       <string>Remark</string>
      </property>
     </widget>
    </item>
    <item row="7" column="1">
     <widget class="QLineEdit" name="remark_label_2">
      <property name="maximumSize">
       <size>
        <width>300</width>
        <height>50</height>
       </size>
      </property>
      <property name="font">
       <font/>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="code_label">
      <property name="maximumSize">
       <size>
        <width>230</width>
        <height>100</height>
       </size>
      </property>
      <property name="sizeIncrement">
       <size>
        <width>50</width>
        <height>0</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Times New Roman</family>
        <pointsize>20</pointsize>
        <weight>9</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(255, 255, 255);
border: 2px solid gray;
border-radius: 10px;
padding: 0 8px;
selection-background-color: darkgray;
font: 75 20pt &quot;Times New Roman&quot;;</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <property name="text">
       <string>PCB Code</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="name_entry">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>50</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>300</width>
        <height>50</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Times New Roman</family>
        <pointsize>15</pointsize>
        <weight>9</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(255, 255, 255);
border: 2px solid gray;
border-radius: 10px;
padding: 0 8px;
selection-background-color: darkgray;
font: 75 15pt &quot;Times New Roman&quot;;</string>
      </property>
      <property name="placeholderText">
       <string>PCB Name</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QComboBox" name="status_combo">
      <property name="maximumSize">
       <size>
        <width>300</width>
        <height>50</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Times New Roman</family>
        <pointsize>15</pointsize>
        <weight>9</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(255, 255, 255);
border: 2px solid gray;
border-radius: 10px;
padding: 0 8px;
selection-background-color: darkgray;
font: 75 15pt &quot;Times New Roman&quot;;</string>
      </property>
      <property name="maxCount">
       <number>100</number>
      </property>
      <item>
       <property name="text">
        <string>Pass</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Fail</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Scrap</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="operator_label">
      <property name="maximumSize">
       <size>
        <width>230</width>
        <height>100</height>
       </size>
      </property>
      <property name="sizeIncrement">
       <size>
        <width>50</width>
        <height>0</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Times New Roman</family>
        <pointsize>20</pointsize>
        <weight>9</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(255, 255, 255);
border: 2px solid gray;
border-radius: 10px;
padding: 0 8px;
selection-background-color: darkgray;
font: 75 20pt &quot;Times New Roman&quot;;</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <property name="text">
       <string>Operator</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="status_label">
      <property name="maximumSize">
       <size>
        <width>230</width>
        <height>100</height>
       </size>
      </property>
      <property name="sizeIncrement">
       <size>
        <width>50</width>
        <height>0</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Times New Roman</family>
        <pointsize>20</pointsize>
        <weight>9</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(255, 255, 255);
border: 2px solid gray;
border-radius: 10px;
padding: 0 8px;
selection-background-color: darkgray;
font: 75 20pt &quot;Times New Roman&quot;;</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <property name="text">
       <string>Status</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
